### ISSUE TITLE
Drag-and-dropping a file onto a dev window opens it in a non-dev window

### ISSUE THREAD

**Issue by 50Wliu** [atom/atom#19240]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: https://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

Drag-and-dropping a file onto a dev window will open it in an existing non-dev window if possible; if no non-dev windows exists, it will create a new one.

### Steps to Reproduce

1. Open a dev window (`atom --dev`)
2. Attempt to drag-and-drop a file onto that dev window

**Expected behavior:** File to be added to the window

**Actual behavior:** File is created in an existing non-dev window if it exists; otherwise a new non-dev window is created

**Reproduces how often:** 100%

### Versions

```
Atom    : 1.38.0-dev-41d934fc0
Electron: 2.0.18
Chrome  : 61.0.3163.100
Node    : 8.9.3
```

### Additional Information

Maybe this line needs adjusting...?
`this.applicationDelegate.open({pathsToOpen})`

/cc @smashwilson

**Comment by smashwilson** [comment]
👍 I can take a look. Does this also happen in safe mode windows?

**Comment by 50Wliu** [comment]
Yep - was just about to comment that!

**Comment by liquidmetalrob** [comment]
This is still a problem.

**Comment by 50Wliu** [comment]
Hi @liquidmetalrob, I no longer see this problem after #19247 - can you please open a new issue so we can triage it further? Thanks!
